sections/02_intro.md

```diff
@@ -1,31 +1,27 @@
 ## Introduction
 
-Biology and medicine are rapidly becoming particularly data-intensive field with
+Biology and medicine are rapidly becoming particularly data-intensive with
 respect to both research and practice. A recent comparison of genomics with
 social media, online videos and other data-intensive scientific disciplines
 suggested that the field of genomics alone would equal or surpass other fields
 in data generation and analysis within the next decade
 [@doi:10.1371/journal.pbio.1002195]. These data present new opportunities, but
-also new challenges. We expect that algorithms to automatically extract
-meaningful patterns and provide sufficient context to enable us to act will be
-required.
+also new challenges. We foresee a great need for algorithms that can automatically extract meaningful patterns while providing sufficient context to act, all within data constrained and privacy critical environments.
 
 Concurrent with this explosive growth in biomedical data, a new class of machine
-learning algorithm has become widespread in the domain of image analysis.
+learning algorithm was initially popularized in the domain of image analysis.
 Computer scientists are now building many-layered neural networks from
-collections of millions of images. In a famous example, scientists from Google
+collections of millions of images. In an early famous example, scientists from Google
 demonstrated that a neural network could learn to identify cats simply by
 watching online videos [@doi:10.1109/ICASSP.2013.6639343]. Such approaches,
 termed deep learning, seem like a solution to the challenge presented by the
 growth of data in biomedicine. Perhaps these algorithms could identify the
-biological "cats" hidden in our data - the patterns that exist but that we don't
-know to look for - and could act on them.
+biological "cats" hidden in our data - the patterns unknown to researcher - and could act on them.
```

Comment:
It's also a bit unclear whether the algorithms or researcher are acting. It was unclear before so feel free to chose. And there's some grammar issues with the bit between the -s